dotty-tensorflow/src/main/scala/me.shadaj.scalapy.tensorflow/example/BidirectionalLSTMExample.scala

```diff
@@ -13,42 +14,41 @@ import scala.language.implicitConversions
 object BidirectionalLSTMExample extends Runnable {
 
   def run() = {
-    val tf = Modules.tensorflow
+    val tf = new TensorFlow()
```

Comment:
Hmm would it be possible to make `TensorFlow` an object rather than a class? Coz logically we rather don't want two instances of `TensorFlow` to co-exist (?)

Reply:
ok, I don't really see a difference but it can be done

Reply:
I think there's a significant difference... a need to create a `TensorFlow` instance in every context where TF's methods are used (or pass the TF instance around) is far less convenient than having a single object that can be imported.

Reply:
Changed in Batch 1